test/supertestSetup.js

```diff
@@ -19,11 +19,11 @@ const auditStub = {
     record: sinon.stub()
 };
 
-const signInServiceStub = {
+const createSignInServiceStub = () => ({
```

Comment:
why the change?

Reply:
Most of the others now use the create method. Having one that is used across tests caused the test suite to hang when I called a reset

Reply:
:+1: